cpg_workflows/inputs.py

```diff
@@ -42,6 +42,8 @@ def get_inputs() -> Cohort | MultiCohort:
 
     # NOTE: When configuring sgs in the config is deprecated, this will be removed.
     if config_retrieve(['workflow', 'input_cohorts'], []):
```

Comment:
`if config_retrieve(['workflow', 'input_cohorts'], []):` can be replaced with `custom_cohort_ids`, this was already set above

Reply:
Also not really relevant to anything, the logical test above can be changed to
```
if (input_datasets := config_retrieve(['workflow', 'input_datasets'], [])) and (custom_cohort_ids := config_retrieve(['workflow', 'input_cohorts'], [])):
    ...
```

and I think that's cool

Reply:
define cool 😂

Reply:
I love a cheeky walrus 🦭